src/main/java/policy/ingest/CreateNodesFromJson.java

```diff
@@ -360,6 +370,43 @@ private Map<String, Integer> createEntityNodesAndRelationships(Node documentNode
         );
     }
 
+    private Map<String, Integer> createAuthNodesAndRelationships(Node documentNode, String authorityName, Transaction tx, Log log) {
+        try {
+            Integer nodesCreated = 0;
+            Integer propertiesSet = 0;
+            Integer relationshipsCreated = 0;
+
+            Node tmp = tx.findNode(Label.label("Entity"), "name", authorityName);
+            if (isNull(tmp)) {
+                Result result = tx.execute("MATCH (n:Entity) WHERE '" + authorityName + "' in split(n.aliases,';') RETURN n");
+                while (result.hasNext()) {
+                    Map<String, Object> row = result.next();
+                    tmp = (Node) row.get("n");
+                }
+            }
+
+            if (isNull(tmp)) {
+                tmp = tx.createNode(Util.labels(Collections.singletonList("Entity")));
+                tmp.setProperty("name", authorityName);
+                nodesCreated++;
+                propertiesSet++;
+            }
```

Comment:
Hey @cWren0110 , if the display_org_s value for a document doesn't match an entity (based on name or any of the aliases) should we make a node for the display_org_s value... Examples of this would be documents with `FAR` or `DFAR` `display_org_s` values... We can remove this snippet if we don't want to create nodes for these (in which case certain documents would have no link back to the authority graph cc: @dakotahavel